setup/quickstart/halyard-gce.md

```diff
@@ -202,10 +202,10 @@ gcloud iam service-accounts keys create $GCS_SERVICE_ACCOUNT_DEST \
 
 ## Part 3: Set Spinnaker configuration
 
-We will install Spinnaker v0.6.0
+We will install Spinnaker v1.0.1
 
 ```bash
-hal config version edit --version 0.6.0
+hal config version edit --version 1.0.1
```

Comment:
You can replace this with `$(hal version latest -q)` -- courtesy of @danielpeach's appengine codelab